src/components/MultiSelect/styles.ts

```diff
@@ -20,24 +16,29 @@ export default styled.article`
   padding: 0;
   margin: 0;
   background: ${ ({ theme }) => theme.colors.grey.light };
+  cursor: pointer;
 
   &.opened {
     border-bottom-color: ${ ({ theme }) => theme.colors.grey.medium };
     box-shadow: 0 -2px 6px 0 ${ ({ theme }) => theme.colors.typography.medium }1F;
     background: white;
 
+    .down {
+      transform: rotate(180deg);
```

Comment:
change to icon up instead of rotating it

Reply:
Done